custom_dc/quickstart.md

```diff
@@ -198,13 +198,13 @@ Note that NL support increases the startup time of your server and consumes more
 
 ## Send an API request
 
-A custom instance can accept REST API requests at the endpoint `/core/api/v2/`. To try it out, here's an example request that returns the same data as in the interactive queries above, using the `observation` API. You can enter this query in your browser to get nice output:
+A custom instance can accept [REST API](/api/rest/v2/index.html) requests at the endpoint `/core/api/v2/`. To try it out, here's an example request that returns the same data as in the interactive queries above, using the `observation` API. You can enter this query in your browser to get nice output:
```

Comment:
nit: perhaps add "here's an example request _you can make to your local Custom DC instance_ that returns..."